### PR TITLE
Doc: Organized the Getting Started section

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
Multiple changes:
- Created a new section "Introduction to Grafana".
- Updated Prometheus and InfuxDB getting started topics.
- Hide getting started with MS SQL Server topic.

### CODE REVIEW

File: docs/sources/getting-started/getting-started-prometheus.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ weight = 300
 
 Prometheus is an open source systems monitoring system for which Grafana provides out-of-the-box support. This topic walks you through the steps to create a series of dashboards in Grafana to display system metrics for a server monitored by Prometheus.
 
-You can also configure a [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) instance to display system metrics without having to host Grafana yourself.
+> **Note:**  You can also configure a [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) instance to display system metrics without having to host Grafana yourself. Grafana offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs)  to help you get started.
```
Comment (GrafanaWriter) [inline docs/sources/getting-started/getting-started-prometheus.md:13]: perhaps simply:
```suggestion
> **Note:**  You can configure a [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) instance to display system metrics without having to host Grafana yourself. A [free forever plan](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) provides 10,000 active series for metrics.
```

File: docs/sources/getting-started/getting-started-prometheus.md [hunk 2]
```diff
@@ -62,3 +62,9 @@ Now that you have a curated list of queries, create [dashboards]({{< relref "../
 The following image shows a dashboard with three panels showing some system metrics.
 
 <img width="850px" src="/static/img/docs/getting-started/simple_grafana_prom_dashboard.png" caption="Prometheus dashboards" >
+
+Here are some resources to learn more:
```
Comment (GrafanaWriter) [inline docs/sources/getting-started/getting-started-prometheus.md:66]:
```suggestion
To learn more:
```

File: docs/sources/introduction/_index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
++++
+title = "Introduction to Grafana"
+weight = 5
+aliases = ["/docs/grafana/latest/guides/what-is-grafana"]
++++
+
+# Introduction to Grafana
+
+Grafana is a complete observability stack that allows you to monitor and analyze mertrics, logs and traces. When monitoring data, an understanding of relevant data and the relationship between them is important. Grafana allows you to seamlessly go from metrics to logs, metrics to traces, and from logs to traces. For more information, refer to [Grafana overview](https://grafana.com/grafana/). Our observability stack has the following products and components.
```
Comment (GrafanaWriter) [inline docs/sources/introduction/_index.md:9]: Can we incorporate any of this content from the Product page? "Grafana allows you to query, visualize, alert on and understand your metrics no matter where they are stored. Create, explore, and share beautiful dashboards with your team and foster a data driven culture."

Comment (GrafanaWriter) [reply docs/sources/introduction/_index.md:9]: maybe we should say "data no matter where it is stored" to better incorporate logs and traces?

File: docs/sources/introduction/oss-details.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
++++
+title = "What is Grafana OSS"
+weight = 5
+aliases = ["/docs/grafana/latest/guides/what-is-grafana"]
++++
+
+# What is Grafana OSS
+
+[Grafana open source software](https://grafana.com/oss/) allows you to query, visualize, alert on, and understand your data no matter where it’s stored. With Grafana you can create, explore and share all of your data through beautiful, flexible dashboards.
```
Comment (GrafanaWriter) [inline docs/sources/introduction/oss-details.md]: This is great. Can we use it in the _index file?

File: docs/sources/introduction/oss-details.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+After you have [installed Grafana]({{< relref "../installation/_index.md" >}}) and set up your first dashboard using instructions in [Getting started with Grafana]({{< relref "../getting-started/getting-started.md" >}}), there are many things you can do next depending on your requirements. For example, if you want to view weather data and statistics about your smart home, then you might create a [playlist]({{< relref "../dashboards/playlist.md" >}}). If you are the administrator for a corporation and are managing Grafana for multiple teams, then you might need to set up [provisioning]({{< relref "../administration/provisioning.md" >}}) and [authentication]({{< relref "../auth/_index.md" >}}).
```
Comment (GrafanaWriter) [inline docs/sources/introduction/oss-details.md]:
```suggestion
After you have [installed Grafana]({{< relref "../installation/_index.md" >}}) and set up your first dashboard using our [Getting started with Grafana]({{< relref "../getting-started/getting-started.md" >}}), what you can do next depends on what you'd like to accomplish. For example, to view weather data and statistics about your smart home, you might create a [playlist]({{< relref "../dashboards/playlist.md" >}}). If you are the administrator for a corporation and managing Grafana for multiple teams, you might need to set up [provisioning]({{< relref "../administration/provisioning.md" >}}) and [authentication]({{< relref "../auth/_index.md" >}}).
```

File: docs/sources/introduction/oss-details.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+The following sections provide an overview of things you might want to do with your Grafana instance and links to product documentation to help you learn more. For more guidance and ideas, check out the [Grafana Community forums](https://community.grafana.com/).
```
Comment (GrafanaWriter) [inline docs/sources/introduction/oss-details.md:13]:
```suggestion
In following sections, you can read an overview of what you can do with your Grafana instance. The overview provides links to product documentation to help you learn more. For more guidance and ideas, check out our [Grafana Community forums](https://community.grafana.com/).
```

**Review by GrafanaWriter — COMMENTED** [review]
Great start!

File: docs/sources/introduction/_index.md (hunk 3 above)
Comment (achatterjee-grafana) [reply docs/sources/introduction/_index.md:9]: I have added your suggestion.

File: docs/sources/introduction/oss-details.md (hunk 6 above)
Comment (achatterjee-grafana) [reply docs/sources/introduction/oss-details.md:13]: Updated content with your suggestion.

**Comment by torkelo** [comment]
If you open these files in vs code with prettier extension installed, and then format each document with prettier it should pass. 

Or run
* yarn install --pure-lockfile 
* yarn run prettier:write

Or: 
* yarn install --pure-lockfile 

then modify each file (just a whitespace at the end), then commit change, the pre-commit hook should format each file with pretttier

**Comment by achatterjee-grafana** [comment]
> If you open these files in vs code with prettier extension installed, and then format each document with prettier it should pass.
> 
> Or run
> 
> * yarn install --pure-lockfile
> * yarn run prettier:write
> 
> Or:
> 
> * yarn install --pure-lockfile
> 
> then modify each file (just a whitespace at the end), then commit change, the pre-commit hook should format each file with pretttier

Thank you, @torkelo.

File: docs/sources/introduction/oss-details.md (hunk 5 above)
Comment (achatterjee-grafana) [reply docs/sources/introduction/oss-details.md]: Incorporated most of the suggestions.

File: docs/sources/introduction/oss-details.md (hunk 4 above)
Comment (achatterjee-grafana) [reply docs/sources/introduction/oss-details.md]: I was trying to avoid duplicate content. Let's keep this as is, can always update the docs later.

**Pull request merged**